### ISSUE TITLE
Dropdown: support Menu classname

### ISSUE THREAD

**Issue by notadamking** [Semantic-Org/Semantic-UI-React#448]
Dropdown.Menu should support passing className as a prop in the same way as Dropdown.Item does, but currently this seems to be missing.

**Comment by levithomason** [comment]
Shorthand props do not support many use cases.  I'm not sure if we should extend the shorthand API since we have the sub component API.  Sub components are there for this reason. They are an escape hatch when shorthand props do not provide what you need.

You can pass any props to the Menu if you use the sub component API.  See this for an example:

http://technologyadvice.github.io/stardust/modules/dropdown#dropdown

**Comment by levithomason** [comment]
It is rather trivial to add a `menuProps` prop for passing more props.  My hesitancy here is that if this were done, all other components' shorthand props would need the same functionality to keep the APIs consistent.

This would mean supporting Message props such as `iconProps`, `contentProps`, `headerProps`, `listProps`, and `itemProps` since it has these shorthand props which map to its sub components.

I am open to other proposals if they are consistent for all components and do not add too much API surface area.  Will re-open in light of those ideas, if any.

**Comment by notadamking** [comment]
@levithomason As it is currently, I do not see any possible way to style the dropdown menu item with inline styles. I would like to be able to change the border radius of the dropdown menu but this is not currently possible.

**Comment by levithomason** [comment]
Per this doc example http://technologyadvice.github.io/stardust/modules/dropdown#dropdown you can put any props on any sub component you like.  Sub components give you complete access to all the markup:

``` js
import React from 'react'
import { Dropdown } from 'stardust'

const DropdownExample = () => (
  <Dropdown text='File'>
    <Dropdown.Menu>
      <Dropdown.Item text='New' />
      <Dropdown.Item text='Open...' description='ctrl + o' />
      <Dropdown.Item text='Save as...' description='ctrl + s' />
      <Dropdown.Item text='Rename' description='ctrl + r' />
      <Dropdown.Item text='Make a copy' />
      <Dropdown.Item icon='trash' text='Move to trash' />
      <Dropdown.Divider />
      <Dropdown.Item text='Download As...' />
      <Dropdown.Item text='Publish To Web' />
      {/* item text can also be defined as children */}
      <Dropdown.Item>E-mail Collaborators</Dropdown.Item>
    </Dropdown.Menu>
  </Dropdown>
)

export default DropdownExample
```

That said, I'd highly recommend customizing a SUI theme that had a different menu item border radius instead of adding inline overrides to your components.  http://semantic-ui.com/usage/theming.html

It may take a little longer to setup, but your code will be much cleaner and your styles will be consistent for all components.

Note, if you use sub components then you need handle `<Dropdown.Item onClick={...} />` yourself. Since the Dropdown is no longer creating items for you, it cannot also attach click event handlers.

**Comment by notadamking** [comment]
I get what you're saying, but I don't think this is a theming issue. Take a look at this

```
import React from 'react'
import { Dropdown } from 'stardust'

const DropdownExample = () => (
  <Dropdown text='File' className='ourDropdown'> /* <---- */
    <Dropdown.Menu className='ourDropdownMenu'> /* <---- */
      <Dropdown.Item text='New' className='ourDropdownItem' /> /* <---- */
      <Dropdown.Item text='Open...' description='ctrl + o' />
      <Dropdown.Item text='Save as...' description='ctrl + s' />
      <Dropdown.Item text='Rename' description='ctrl + r' />
      <Dropdown.Item text='Make a copy' />
      <Dropdown.Item icon='trash' text='Move to trash' />
      <Dropdown.Divider />
      <Dropdown.Item text='Download As...' />
      <Dropdown.Item text='Publish To Web' />
      {/* item text can also be defined as children */}
      <Dropdown.Item>E-mail Collaborators</Dropdown.Item>
    </Dropdown.Menu>
  </Dropdown>
)

export default DropdownExample
```

I have added 3 className props to your example to show what I mean. When react renders the DropdownExample, the Dropdown and Dropdown.Item classNames will be rendered on their respective dom elements (i.e. `<div class="ui ourDropdown dropdown">` and `<div class="item ourDropdownItem">`) but Dropdown.Menu's rendered dom element doesn't receive the passed className like the others (i.e. `<div class="menu transition visible">`). Should Dropdown.Menu's api not be consistent with Dropdown.Item's?

**Comment by levithomason** [comment]
Ah, apologies for the prolonged misunderstanding.  Thanks for sticking it out. I've confirmed the bug on master.  Patch release coming to resolve it.

**Comment by levithomason** [comment]
Released in `stardust@0.38.1`
